src/Extensions.cs

```diff
@@ -113,6 +113,11 @@ public static string[] ToWords(this string value)
             return WordSplitPattern.Split(value).Where(s => !string.IsNullOrEmpty(s)).ToArray();
         }
 
+        public static string ToIdiomaticShortName(this string value)
+        {
+            return CodeNamerGo.Instance.GetVariableName(string.Concat(value.ToPhrase().Split(new char[]{' '}).Select(w => w[0])));

```

Comment:
I think this could be rewritten to be a little bit easier to read. Maybe,

``` C#
public static string ToIdiomaticShortName(this string longName)
{
    var initials = from word in longName.ToPhrase().Split(new []{' '})
                         select word[0];
    var acronym = string.Concat(initials);
    return CodeNamerGo.Instance.GetVariableName(acronym);
}
```

Reply:
To throw another option into the mix, I wrote the follow function for creating vars from pascal-cased strings.
```
public static string ToShortVarName(this string name)
        {
            var sb = new StringBuilder();
            for (int i = 0; i < name.Length; ++i)
            {
                if (char.IsUpper(name[i]))
                {
                    sb.Append(char.ToLowerInvariant(name[i]));
                }
            }
            return sb.ToString();
        }
```

Reply:
Fixed